<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitsDialog</class>
 <widget class="QDialog" name="UnitsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rental Units</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QLabel" name="lblTenant">
       <property name="text">
        <string>Tenant Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="4">
      <widget class="QLineEdit" name="txtElecBillAcc"/>
     </item>
     <item row="3" column="2" colspan="4">
      <widget class="QLineEdit" name="txtWaterBillAc"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="txtOccupied">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="cmdEndOccupation">
       <property name="text">
        <string>End Occupation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLineEdit" name="txtTenantName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Water Bill Acc:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Occupied:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Unit No:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>No. Of Rooms:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Elec Bill Acc:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QDoubleSpinBox" name="spSqFt"/>
     </item>
     <item row="0" column="2" colspan="5">
      <widget class="QLineEdit" name="txtUnitNo"/>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sq Ft:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="QSpinBox" name="spNoOfRooms"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>63</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtUnitNo</tabstop>
  <tabstop>spNoOfRooms</tabstop>
  <tabstop>txtOccupied</tabstop>
  <tabstop>txtWaterBillAc</tabstop>
  <tabstop>txtElecBillAcc</tabstop>
  <tabstop>txtTenantName</tabstop>
  <tabstop>spSqFt</tabstop>
  <tabstop>cmdEndOccupation</tabstop>
  <tabstop>cmdSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
